src/stores/TxStore.js

```diff
@@ -33,15 +30,13 @@ class TxStore {
           data
         }).on('transactionHash', (hash) => {
           console.log('txHash', hash)
-          this.txHashToIndex[hash] = index;
-          this.txs[index] = {status: 'pending', name: `Sending ${to} ${value}`, hash}
           this.alertStore.setLoadingStepIndex(1)
           addPendingTransaction()
           this.getTxReceipt(hash)
         }).on('error', (e) => {
           if(!e.message.includes('not mined within 50 blocks')){
```

Comment:
During the testing it was found that even if the error appears, there is no notification to the user that web3 stops waiting for the transaction to be included to a block.
![image](https://user-images.githubusercontent.com/20793260/46808297-af8db400-cd74-11e8-83b1-8821718a5e84.png)
This could confuse the user - it will never know that he needs to re-send the transaction.

Reply:
This behavior is on purpose, on https://github.com/poanetwork/bridge-ui/issues/83 we decided to ignore this error and use `fast` speed from oracle. But I think we can open a new issue to discuss which is the best way to present this to the user and give the option to re-send the transaction

Reply:
Ok. I will open one.

Reply:
Created https://github.com/poanetwork/bridge-ui/issues/144 to address this issue.